app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+# Top-level documentation comment for Post module
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def update_user_posts_counter
+    author.update(posts_counter: author.posts.count)
+  end
+
+  def recent_comments(num)
+    comments.order(created_at: :desc).limit(num)
+  end
```

Comment:
- [ ]  @Uthmanbello great job adding the recent_comments method, however, the method should only return the 5 most recent comments by default, in your case, you have made the number of comments a variable kindly make it a fixed value of 5.
![image](https://user-images.githubusercontent.com/61408860/236511765-f2e2db27-b0dd-4815-9108-de5e873d9f16.png)